tests/transform/test_biomarkers.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import pandas as pd
+import pytest
+
+from agoradatatools.etl.transform import biomarkers
+
+"""
+Test Cases to-do:
+- What happens if there are two identical entries?
+"""
+
+
+class TestTransformBiomarkers:
+    data_files_path = "tests/test_assets/biomarkers"
+    pass_test_data = [
+        (  # Pass with good real data
+            "biomarkers_good_input.csv",
+            "biomarkers_good_output.json",
+        ),
+        (
+            # Pass with good fake data
+            "biomarkers_good_test_input.csv",
+            "biomarkers_good_test_output.json",
+        ),
+        (
+            # Pass with duplicated data
+            "biomarkers_duplicated_input.csv",
+            "biomarkers_duplicated_output.json",
+        ),
+    ]
+    pass_test_ids = [
+        "Pass with good real data",
+        "Pass with good fake data",
+        "Pass with duplicated data",
+    ]
+    fail_test_data = [
+        # No failure cases for this transform
```

Comment:
In your transform you have 2 error conditions: A TypeError and a ValueError. Can you make some test data that will force each of these to happen in a test? `test_proteomics_distribution_data.py` has an example of failure condition code that checks for different error types depending on input.

Reply:
Actually now that I look at it, I'm not sure you _can_ trigger the `TypeError` condition, since the input _has_ to be a data frame by the time it gets to your transform. If it wasn't a DF, that means one of the earlier functions (load or standardize) would have failed first. In which case, it might be worth just removing that error check. `test_genes_biodomains.py` has a simpler, one-error-type failure case you can look at in that case.